content/en/docs/concepts/workloads/pods/_index.md

```diff
@@ -191,6 +191,30 @@ details are abstracted away. That abstraction and separation of concerns simplif
 system semantics, and makes it feasible to extend the cluster's behavior without
 changing existing code.
 
+## Pod update and replacement
+
+As mentioned in the previous section, when the Pod template for a workload
+resource is changed, the controller will create new Pods based on the updated
+template instead of updating or patching the existing Pods.
+
+Kubernetes doesn't prevent you from managing Pods directly. It is possible to
+update a Pod in place. However, the Pod update operation has some limitations:
+
+- Most of the metadata about a Pod is immutable. For example, you cannot
+  change the `namespace`, `name`, `uid`, `creationTimestamp` fields, you can
+  only increment the `generation` field if you want to change it.
+- If the `metadata.deletionTimestamp` is set, no new entry can be added to the
+  `metadata.finalizers` list.
+- Pod updates may not change fields other than `spec.containers[*].image`,
+  `spec.initContainers[*].image`, `spec.activeDeadlineSeconds` or
+  `spec.tolerations`. Only adding new item into `spec.tolerations` is
+  permitted.
+- When updating the `spec.activeDeadlineSeconds` field, two types of updates
+  are allowed:
```

Comment:
@sftim suggestions look good. Does the `nil` setting imply that the field is unset or possibly set to 0, ""?
You can change the `spec.activeDeadlineSeconds` field by:
- modifying the field from `nil` to a positive number, or
- modifying the field from a positive number ...

Reply:
nil is unset.

Reply:
nil is a bit surprising. On the wire it's JSON and the field would be _I think_ `null` or just not present. If you used `kubectl edit replicaset/foo` (say) then you'd get YAML and again there's either `null` or you omit the field.

In the source code for the API server, it's `nil`. But: Kubernetes tries to be implementation neutral; if someone wants to write a 100% conformant Kubernetes API server in Python, more strength to them.